mne/cuda.py

```diff
@@ -19,6 +19,48 @@
 _cuda_capable = False
 
 
+def get_shared_mem(
+    shape,
+    dtype=np.float64,
+    strides=None,
+    order="C",
+    stream=0,
+    portable=False,
+    wc=True,
+):
+    """Get shared memory space to avoid copying from cpu to gpu when possible.
+
+    Allocate a mapped ndarray with a buffer that is pinned and mapped on
+    to the device. Similar to np.empty()
+
+    Parameters
+    ----------
+    portable: bool
+        a boolean flag to allow the allocated device memory to be
+              usable in multiple devices.
+    wc: bool
+        a boolean flag to enable writecombined allocation which is faster
+        to write by the host and to read by the device, but slower to
+        write by the host and slower to write by the device.
+
+    Returns
+    -------
+    a mapped array: np.ndarray
+        An array to be passed into cupy.asarray, which does not copy if shared memory is already allocated.
```

Comment:
Most CIs will not run until style checks pass, and this E501 is problematic so I'll commit this. Feel free to force-push over (or pull the change) if you want
```suggestion
        An array to be passed into cupy.asarray, which does not copy if
        shared memory is already allocated.
```